cgi-bin/DW/Controller/Entry.pm

```diff
@@ -1203,6 +1209,8 @@ sub _persist_props {
     my ( $u, $form ) = @_;
 
     return unless $u;
+    
+    $u->displaydate_check( $form->{update_displaydate} ? 1 : 0);
```

Comment:
style nitpick: consistent whitespace please (don't need it after the parentheses above)